tests/test_changes.bats

```diff
@@ -29,6 +31,19 @@ function teardown {
   unset_current_state
 }
 
+@test "run 'changes' on one file with no file changed" {
+  local password=$(test_user_password "$TEST_DEFAULT_USER")
+  run git secret changes -d "$TEST_GPG_HOMEDIR" -p "$password" "$FILE_TO_HIDE"
+
+  echo "$output" | sed "s/^/# '$BATS_TEST_DESCRIPTION' output: /" >&3
+
+  [ "$status" -eq 0 ]
+
+  local num_lines=$(echo "$output" | wc -l)
+  echo "# '$BATS_TEST_DESCRIPTION': num lines is $num_lines" >&3
+  [[ "num_lines" -eq 1 ]]
```

Comment:
Just to be clear. How does this work? Looks like an unknown syntax to me 🙂 

Should not it be `[[ "$num_lines" -eq 1 ]]`?

Reply:
yes, typo